### PR TITLE
[CI] Use Jenkins pipeline and ansible playbooks

### PR DESCRIPTION
**Pull request by rpitonak**
Jobs:
https://ci.centos.org/job/conu-master/
https://ci.centos.org/job/conu-pr/

### CODE REVIEW

File: files/install-openshift.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  hosts: all
+  tasks:
+  - name: Install rpms.
+    yum:
+      name:
+      - git
+      - docker
+      state: present
+  - name: Disable SELinux
+    command: setenforce 0
```
Comment (jpopelka) [inline files/install-openshift.yaml]: or with [selinux module](https://docs.ansible.com/selinux_module.html)
```yaml
selinux:
  policy: targeted
  state: permissive
```

File: files/install-openshift.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  - name: Disable firewall
+    command: systemctl stop firewalld # firewall on CentOS does not allow docker login into OpenShift registry
+  - name: Start docker deamon
+    command: systemctl start docker
```
Comment (jpopelka) [inline files/install-openshift.yaml]: https://docs.ansible.com/systemd_module.html
```yaml
  systemd:
    state: started
    name: docker
```

**Comment by TomasTomecek** [comment]
[test]

**Comment by TomasTomecek** [comment]
even though some podman tests are failing, I'm merging and will try to fix them in my PR

**Pull request merged**